### PR TITLE
Small portrait improving

### PR DESCRIPTION
**Pull request by karnedi**
## Changelog:
- Added undead portraits of the night elves and highborne.
- The high elves become gold-eyed if they pick paladin and priest classes.

# How to test:
A few console commands:
- `event WCCLS.301` to start paladin training.
- `event WCCLS.400` to finish paladin training (you must start it first).
- `event WCCLS.401` to start priest training.
- `event WCCLS.500` to finish priest training (you must start it first).
- `event WCPOD.5013` to become undead.

### CODE REVIEW

File: common/graphicalculturetypes/wc_graphicalculturetype.txt [hunk 1]
```diff
@@ -493,4 +493,13 @@ nightelffelgfx = {
 }
 highbornefelgfx = {
 	skin_tone = 0
+}
+nightelfundeadgfx = {
```
Comment (zumbak04) [inline common/graphicalculturetypes/wc_graphicalculturetype.txt:497]: To make race traits work correct you need to add it in `has_nightelf_gfx_trigger` and `has_undead_gfx_trigger`.

File: common/graphicalculturetypes/wc_graphicalculturetype.txt [hunk 2]
```diff
@@ -493,4 +493,13 @@ nightelffelgfx = {
...
+	skin_tone = 0
+}
+highborneundeadgfx = {
```
Comment (zumbak04) [inline common/graphicalculturetypes/wc_graphicalculturetype.txt:500]: To make race traits work correct you need to add it in `has_highborne_gfx_trigger` and `has_undead_gfx_trigger`.

File: common/graphicalculturetypes/wc_graphicalculturetype.txt [hunk 3]
```diff
@@ -493,4 +493,13 @@ nightelffelgfx = {
...
+	skin_tone = 0
+}
+highelflighttouchedgfx = {
```
Comment (zumbak04) [inline common/graphicalculturetypes/wc_graphicalculturetype.txt:503]: To make race traits work correct you need to add it in `has_highelf_gfx_trigger`.

File: common/cultures/culture_undead.txt [hunk 4]
```diff
@@ -263,6 +263,27 @@ undead_group = {
...
+		
+		color = { 0.63 0 0.98 }
+		
+		used_for_random = no
+		allow_in_ruler_designer = no
+		
+		modifier = swarm_culture_modifier
+	}
+	highborne_undead = {
+		graphical_cultures = { highelflighttouchedgfx westerngfx }
```
Comment (zumbak04) [inline common/cultures/culture_undead.txt:280]: highelflighttouchedgfx? Are you sure?

File: interface/portraits/portraits_nightelf.gfx [hunk 5]
```diff
@@ -654,6 +654,500 @@ spriteTypes = {
 			{ 200 255 30 } #Yellowish Green
 		}
 		
+		headgear_that_hides_hair = { }
+	}
+	### nightelf FEMALE ####
+
+	# nightelf female
+	portraitType = {
+		name = "PORTRAIT_nightelfgfx_female"
```
Comment (zumbak04) [inline interface/portraits/portraits_nightelf.gfx:663]: It doesn't look like an undead portrait.

File: interface/portraits/portraits_highborne.gfx [hunk 6]
```diff
@@ -559,6 +560,285 @@ spriteTypes = {
...
+			"GFX_character_imprisoned:p6"
+			"GFX_being_overlay:p20"
+			"GFX_player_overlay:p11"
+		}
+
+		hair_color_index = 8
+		hair_color = { # dark, base, highlight
+			#Night elf colors
+			{ 0 0 0 } {  190 200 210 } { 255 255 255 } # White
+		}
```
Comment (zumbak04) [inline interface/portraits/portraits_highborne.gfx:788]: Undead should use "old" colors.
```suggestion
		hair_color = { # dark, base, highlight
			#Night elf colors
			{ 0 0 0 } {  237	240	243 } { 255 255 255 } # Old white
		}
```

**Comment by karnedi** [comment]
I fixed portraits of the highborn

And The helmet on female quel'dorei  through the hair not a bug this is how it should be

**Pull request merged**